aptos-move/framework/aptos-stdlib/sources/data_structures/big_vector.spec.move

```diff
@@ -100,20 +117,14 @@ spec aptos_std::big_vector {
             spec_at(v, idx) == spec_at(old(v), idx);
     }
 
-    spec append<T: store>(lhs: &mut BigVector<T>, other: BigVector<T>) {
-        pragma verify=false;
-    }
-
-    spec remove<T>(v: &mut BigVector<T>, i: u64): T {
-        pragma verify=false;
-    }
-
     spec reverse<T>(v: &mut BigVector<T>) {
         pragma verify=false;
+        // ensures forall i in 0..v.end_index: spec_at(v, i) == spec_at(old(v), v.end_index - (i + 1));
     }
 
     spec index_of<T>(v: &BigVector<T>, val: &T): (bool, u64) {
-        pragma verify=false;
+        ensures (result_1 == true) ==> (spec_at(v, result_2) == val);
+        // ensures (result_1 == false) ==> (forall i in 0..v.end_index: spec_at(v, i) != val);
```

Comment:
Proving this assertion will complete the specification for the index_of function. Writing the loop invariant for this specification is challenging.